core/modules/book/book.module

```diff
@@ -892,6 +892,10 @@ function book_node_presave(Node $node) {
   if (empty($node->nid)) {
     $node->book['mlid'] = NULL;
   }
+  // clean up book cache if nessecary
```

Comment:
Throughout this PR, could you make comments sentence-case and end with a period? https://api.backdropcms.org/php-standards#comments